playbooks/openstack/openshift-cluster/update.yml

```diff
@@ -1,4 +1,102 @@
 ---
+- name: Launch instance(s)
+  hosts: localhost
+  connection: local
+  gather_facts: no
+  vars_files:
+  - vars.yml
+  tasks:
+  - fail:
+      msg: "Deployment type not supported for OpenStack provider yet"
+    when: deployment_type == 'online'
+
+  # TODO: Write an Ansible module for dealing with HEAT stacks
+  #       Dealing with the outputs is currently terrible
+
+  - name: Check OpenStack stack
+    command: 'heat stack-show openshift-ansible-{{ cluster_id }}-stack'
+    register: stack_show_result
+    changed_when: false
+    failed_when: stack_show_result.rc != 0 and 'Stack not found' not in stack_show_result.stderr
+
+  - name: Update OpenStack Stack
```

Comment:
There’s a lot of redundancy between `launch.yml` and `update.yml`.
I wish we could keep only one file for those two with a parameter so that:
* In “launch” mode, we do a `heat stack-create` (“launch” will fail if the cluster_id corresponds to an already existing cluster)
* In “update” mode, we do a `heat stack-update` (“update” will fail in the cluster_id doesn’t correspond to an already existing cluster)